modules/openapi-generator/src/test/java/org/openapitools/codegen/DefaultCodegenTest.java

```diff
@@ -836,21 +836,6 @@ private CodegenProperty codegenPropertyWithXEnumVarName(List<String> values, Lis
         return objs;
     }
 
-    @Test
-    public void objectQueryParamIdentifyAsObject() {
```

Comment:
NOTE: Removing this test/functionality would be considered a breaking change and would need to be documented as a regression.